includes/classes/order.php

```diff
@@ -267,41 +267,47 @@ function cart() {
     $customer_address_query = "select c.customers_firstname, c.customers_lastname, c.customers_telephone,
                                     c.customers_email_address, ab.entry_company, ab.entry_street_address,
                                     ab.entry_suburb, ab.entry_postcode, ab.entry_city, ab.entry_zone_id,
-                                    z.zone_name, co.countries_id, co.countries_name,
+                                    z.zone_name, co.countries_id, con.countries_name,
```

Comment:
In this file you use `con.` as the alias, but in all the other files you've touched you are using `cn.`
I suggest changing this file to match by using `cn.`